src/components/CreateProposal/CreateProposalContainer.tsx

```diff
@@ -50,8 +51,8 @@ class CreateProposalContainer extends React.Component<IProps, null> {
           : schemeName === "SchemeRegistrar" ?
             <CreateSchemeRegistrarProposal daoAvatarAddress={daoAvatarAddress} handleClose={this.goBack.bind(this)} />
           : schemeName === "GenericScheme" ?
-            <CreateGenericSchemeProposal daoAvatarAddress={daoAvatarAddress} handleClose={this.goBack.bind(this)} />
-          : ""
+            <CreateDutchXProposal daoAvatarAddress={daoAvatarAddress} handleClose={this.goBack.bind(this)} />
```

Comment:
Given that we are tryuing to implement a generic solution here, perhaps we should rename these components and remove any references to DutchX...